stdlib/sampledata/sampledata.flux

```diff
@@ -0,0 +1,310 @@
+,,1,2021-01-01T00:00:10Z,t2,4.97
+,,1,2021-01-01T00:00:20Z,t2,-3.75
+,,1,2021-01-01T00:00:30Z,t2,19.77
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "13.86") + "
+,,1,2021-01-01T00:00:50Z,t2,1.86
+"
+
+_string = (includeNull=false) => "
+#group,false,false,false,true,false
+#datatype,string,long,dateTime:RFC3339,string,string
+#default,_result,,,,
+,result,table,_time,tag,_value
+,,0,2021-01-01T00:00:00Z,t1,smpl_g9qczs
+,,0,2021-01-01T00:00:10Z,t1," + (if includeNull then "" else "smpl_0mgv9n") + "
+,,0,2021-01-01T00:00:20Z,t1,smpl_phw664
+,,0,2021-01-01T00:00:30Z,t1," + (if includeNull then "" else "smpl_guvzy4") + "
+,,0,2021-01-01T00:00:40Z,t1," + (if includeNull then "" else "smpl_5v3cce") + "
+,,0,2021-01-01T00:00:50Z,t1,smpl_s9fmgy
+,,1,2021-01-01T00:00:00Z,t2," + (if includeNull then "" else "smpl_b5eida") + "
+,,1,2021-01-01T00:00:10Z,t2,smpl_eu4oxp
+,,1,2021-01-01T00:00:20Z,t2,smpl_5g7tz4
+,,1,2021-01-01T00:00:30Z,t2,smpl_sox1ut
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "smpl_wfm757") + "
+,,1,2021-01-01T00:00:50Z,t2,smpl_dtn2bv
+"
+
+_bool = (includeNull=false) => "#group,false,false,false,true,false
+#datatype,string,long,dateTime:RFC3339,string,boolean
+#default,_result,,,,
+,result,table,_time,tag,_value
+,,0,2021-01-01T00:00:00Z,t1,true
+,,0,2021-01-01T00:00:10Z,t1," + (if includeNull then "" else "true") + "
+,,0,2021-01-01T00:00:20Z,t1,false
+,,0,2021-01-01T00:00:30Z,t1," + (if includeNull then "" else "true") + "
+,,0,2021-01-01T00:00:40Z,t1," + (if includeNull then "" else "false") + "
+,,0,2021-01-01T00:00:50Z,t1,false
+,,1,2021-01-01T00:00:00Z,t2," + (if includeNull then "" else "false") + "
+,,1,2021-01-01T00:00:10Z,t2,true
+,,1,2021-01-01T00:00:20Z,t2,false
+,,1,2021-01-01T00:00:30Z,t2,true
+,,1,2021-01-01T00:00:40Z,t2," + (if includeNull then "" else "true") + "
+,,1,2021-01-01T00:00:50Z,t2,false
+"
+
+// float returns a sample data set with float values.
+//
+// ## Parameters
+//
+// - `includeNull` indicates whether or not to include null values in the returned dataset.
+//   Default is `false`.
+//
+// ## Output basic sample data with float values
+//
+// ```
+// import "sampledata"
+//
+// sampledata.float()
+// ```
+// 
+// ## Output data
+// 
+// | tag | _time                | _value |
+// | :-: | :------------------- | -----: |
+// | t1  | 2021-01-01T00:00:00Z |  -2.18 |
+// | t1  | 2021-01-01T00:00:10Z |  10.92 |
+// | t1  | 2021-01-01T00:00:20Z |   7.35 |
+// | t1  | 2021-01-01T00:00:30Z |  17.53 |
+// | t1  | 2021-01-01T00:00:40Z |  15.23 |
+// | t1  | 2021-01-01T00:00:50Z |   4.43 |
+
+// | tag | _time                | _value |
+// | :-: | :------------------- | -----: |
+// | t2  | 2021-01-01T00:00:00Z |  19.85 |
+// | t2  | 2021-01-01T00:00:10Z |   4.97 |
+// | t2  | 2021-01-01T00:00:20Z |  -3.75 |
+// | t2  | 2021-01-01T00:00:30Z |  19.77 |
+// | t2  | 2021-01-01T00:00:40Z |  13.86 |
+// | t2  | 2021-01-01T00:00:50Z |   1.86 |
+// 
+float = (includeNull=false) => {
```

Comment:
Do we consider the specific data returned to be part of the API of the function? I.e. if the data changed would we consider that a breaking change? I think we would have too because otherwise examples could break or any test cases that use the data.

Reply:
So is the idea here that we can have completely self-contained examples in  our docs that the user could copy and paste and run in the UI or CLI (or maybe even in an interactive text box right in the docs themselves)?

Reply:
Yep, that's exactly what they're designed for.